python/gtsam/specializations/slam.h

```diff
@@ -17,3 +17,4 @@ py::bind_vector<
 py::bind_vector<
     std::vector<boost::shared_ptr<gtsam::BetweenFactor<gtsam::Pose2> > > >(
     m_, "BetweenFactorPose2s");
+py::bind_vector<std::vector<gtsam::Rot3>>(m_, "Rot3Vector");
```

Comment:
Are you sure you don't need aligned allocator?

Reply:
Thanks for the review @ProfFan and @akshay-krishnan.

Let me check about aligned allocator. Btw I don't love the name `Rot3Vector` so much ... sounds like a vector representation of a Rot3. But `Rot3Set` would not be consistent with `Pose3Vector` nomenclature. So I guess it works.